rose/server/game.py

```diff
@@ -96,9 +98,10 @@ def drive_player(self, name, info):
         self.players[name].response_time = info.get('response_time', 1.0)
 
     def print_stats(self):
-        log.info('Stats')
+        string = 'Stats' + os.linesep
         for i, p in enumerate(sorted(self.players.values())):
-            log.info('%d  %10s  row:%d  score:%d', i + 1, p.name, p.y, p.score)
+            string += '%d  %10s  row:%d  score:%d'.format(i + 1, p.name, p.y, p.score)
```

Comment:
I think we better keep the current string formatting. The good old % placeholder works in every language, and I don't have time to look up in python docs how use the "better" format of the year :-)

Also, we are missing a newline after the first line, but note that we do not want a new line after the second line.

The best way to do this would be:

    lines = ["Stats:"]
    for ...:
        line = ...
        lines.append(line)
    
    log.info("%s", os.linesep.join(lines))